specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/examples/GetConsoleDisabledSuccessExample.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "api-version": "2018-07-01-preview",
+        "subscriptionId": "00000000-00000-0000-0000-000000000000",
+        "default": "default"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "disabled": "true"
```

Comment:
According to spec `GetDisabledResult`, it should be `true` (JSON Boolean), not `"true"` (JSON string).